workers/unity/Packages/io.improbable.gdk.core/Systems/EntitySystem.cs

```diff
@@ -1,61 +1,127 @@
-using System.Collections.Generic;
+using System;
+using Unity.Collections;
 using Unity.Entities;
-using Unity.Profiling;
+using Unity.Jobs;
 
 namespace Improbable.Gdk.Core
 {
     [DisableAutoCreation]
     [AlwaysUpdateSystem]
     [UpdateInGroup(typeof(SpatialOSReceiveGroup.InternalSpatialOSReceiveGroup))]
-    [UpdateBefore(typeof(SpatialOSReceiveSystem))]
-    public class EntitySystem : ComponentSystem
+    [UpdateAfter(typeof(SpatialOSReceiveSystem))]
```

Comment:
Do the systems that rely on running AFTER `EntitySystem` have been updated to keep this order change in mind?

Reply:
After `EntitySystem` previously or now? Using `Ctrl-Shift-F` nothing has a UpdateBefore or After `EntitySystem`